vllm/distributed/eplb/eplb_state.py

```diff
@@ -147,7 +153,24 @@ class EplbState:
     """
     Interval for expert rearrangement steps.
     This is a constant and is taken from the config.
-    """
+    """ 
+    layer: int = 0
+
+    ep_buffer_ready: bool = False
+
+    buffer_lock: threading.Lock = threading.Lock()
```

Comment:
This should probably also use `default_factory` so that each instance of `EplbState` gets a lock?